aioxmpp/muc/service.py

```diff
@@ -39,6 +39,21 @@
 from . import xso as muc_xso
 
 
+def _extract_one_pair(body):
```

Comment:
Unless the MUC service actively introduces empty ``<body/>`` for xml:lang values we did not send, I don’t think that matters. Also see the discussion below.